src/hotspot/os_cpu/linux_ppc/javaThread_linux_ppc.cpp

```diff
@@ -24,6 +24,7 @@
  */
 
 #include "memory/metaspace.hpp"
+#include "os_linux_ppc.inline.hpp"
```

Comment:
So you need to reach `os::Linux::ucontext_get_sp`, right? Then the header that contains it, is a generic `os_linux.hpp`. You get it transitively by including `os_linux_ppc.inline.hpp`, but there is no need to rely on this transitivity.